addons/clickhouse/configs/ch_keeper_00_default_overrides.xml.tpl

```diff
@@ -30,14 +30,14 @@
           <raft_logs_level>warning</raft_logs_level>
       </coordination_settings>
       <raft_configuration>
-      {{- if eq $.TLS_ENABLED "true" }}
+      {{- if and (index . "TLS_ENABLED") (eq .TLS_ENABLED "true") }}
```

Comment:
render failed with nest scope, define a var to resolve it

Reply:
Fixed